spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/registry/DataFlowAppRegistryPopulator.java

```diff
@@ -27,34 +30,39 @@
  *
  * @author Mark Fisher
  */
-public class DataFlowUriRegistryPopulator implements InitializingBean {
+public class DataFlowAppRegistryPopulator implements InitializingBean, ResourceLoaderAware {
```

Comment:
I see that this class is not being used and there is no explicit way to specify the `import` of apps list during the SCDF server startup now. Am I missing something?